sdk/auth/azure_cli_authorizer.go

```diff
@@ -28,11 +36,15 @@ type AzureCliAuthorizerOptions struct {
 	// used for Resource Manager when auxiliary tenants are needed.
 	// e.g. https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/authenticate-multi-tenant
 	AuxTenantIds []string
+
+	// EnableCaching permits module-scoped caching of the parsed results of `az` command invocations, and can be set to improve
+	// performance when instantiating multiple authorizers, at the cost of detectability during long-running applications.
+	EnableCaching bool
```

Comment:
Is there a benefit to making caching configurable? This feels like something we'd always want on?

```suggestion
```

Reply:
Sounds good to me, I've refactored accordingly 👍